pom.xml

```diff
@@ -112,7 +112,7 @@
     <dependency>
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
-      <version>2.8.0</version>
```

Comment:
Please give a short reason. Either inherit it from somewhere or add a comment.
Plugins are seperated by classloaders as far as I remember, so a dependency convergence should not hurt (aside from a few KiB of jars).

Reply:
check the dependency convergence report. It's not just the plugins but shared libraries such as maven-archiver.